frontend/src/components/Card.tsx

```diff
@@ -1,22 +1,59 @@
 import "./Card.scss"
+import React, {useState} from "react";
+import {Button} from "@mui/material";
+
 
 interface cardProps {
     key: string
     name: string
     id: string
     animal_type: string
     image_link: string
+    latin_name: string
 }
 
-export default function Card(props: cardProps){
-    return(
-        <div className="card">
-            <h3>{props.name}</h3>
-            <img className="image" src={props.image_link} alt="Ein Bild"/>
-            <ul>
-                <li>{props.animal_type}</li>
-                <li>{props.id}</li>
-            </ul>
+export default function Card(props: cardProps) {
+    const {animal_type, image_link} = props
+    const [text, setText] = useState<string>('');
+    const language: string = "de-de";
+    const standardtext: string ="mit welchem Buchstaben beginnt mein Name, ich heiße";
+    //todo: set key later in environment
+    const key: string = "a7aae25de0b446c7adc2571316a7ddfc&";
+
+    /**
+     * returns a new string with the first word of the
+     * @param sentence
+     * */
+    function getFirstWord (sentence: string):string {
+        if (sentence !== undefined && sentence.includes(" ")) {
+            return sentence.split(" ")[0];
+        } else {
+            return sentence;
+        }
```

Comment:
```suggestion
        return sentence.split(" ")[0];
```

Reply:
why delete the check?

Reply:
sentence is of type string, so it cant be undefined.
have a look at what .split() returns when the seperator is not in the given string.